tests/D2L.CodeStyle.Analyzers.Test/Async/Generator/AsyncToSyncMethodTransformerTests.cs

```diff
@@ -319,6 +319,15 @@ internal sealed class AsyncToSyncMethodTransformerTests {
 		Assert.AreEqual( @"[Blocking] Foo Bar() { var ct = new CancellationToken(); return Foo( 1,""test"" ); }", actual.Value.ToFullString() );
 	}
 
+	[Test]
+	public void ParenthesizedAnonymousCreationLambda() {
+		var actual = Transform( @"[GenerateSync] Task BarAsync() { () => new { BazAsync = TaskCreator.NewTask, Quux = id.ToStringAsync( ""N"" ) } }" );
+
+		Assert.IsTrue( actual.Success );
+		Assert.IsEmpty( actual.Diagnostics );
+		Assert.AreEqual( @"[Blocking] void Bar() { () => new { Baz = TaskCreator.NewTask,Quux = id.ToString( ""N"" ) } }", actual.Value.ToFullString() );
+	}

```

Comment:
I don't believe `BarAsync` is currently a valid example? It's not returning a task anywhere.